### DISCUSSION TITLE
Version 0.26

### DISCUSSION THREAD

**Discussion by NTDLS** [NTDLS/Katzebase#101] [Announcements]
@ingted @AnibalYeh check out the new release.

I didn't break anything this time except for the tests, but this might be one of the largest updates yet:

https://github.com/NTDLS/Katzebase/releases/tag/0.26.0

**Comment by NTDLS** [comment]
Hope you are doing ok man! 😃

**Reply by ingted** [reply]
Oops!! I tried to have a generic version of Katzebase to support array & struct for our project recently, however I found it was too difficult to catch up the development progress with the main stream... So I stepped back and implemented [this](https://github.com/ingted/KServer/blob/Fatzebase/fstring/PB.fsx) (a toy version persisted concurrent dictionary supported array & struct). And it came up with another serialization issue, so I had to fix [that](https://github.com/mbraceproject/FsPickler/issues/128#issuecomment-2430539182). Oh my god...


It was really tired. Anyway I will come back later and since I don't need the array and struct support anymore, I could focus at just the unit test!! Just few days! 

I noticed there were many useful features you launched in these days, so cool!!
